develop-docs/sdk/telemetry/traces/index.mdx

```diff
@@ -128,6 +128,22 @@ Header Format:
 
 The full spec is available in the [W3C Trace Context](https://www.w3.org/TR/trace-context/) specification.
 
+### `traceIgnoreStatusCodes`
+
+This must be a sequence of integers, denoting HTTP status codes.
+
+The option only applies to incoming requests, and therefore should only be implemented in server SDKs.
+
+The SDK should honor this option by inspecting the [`http.response.status_code`](https://opentelemetry.io/docs/specs/semconv/registry/attributes/http/#:~:text=1437-,http.response.status_code,-int) attribute on each transaction/root span when it's finished.
+If the value of this attribute matches one of the status codes in `traceIgnoreStatusCodes`, the SDK must set the transaction's [sampling decision](https://develop.sentry.dev/sdk/telemetry/traces/#sampling) to `not sampled`.
+
+The SDK should log a debug level message (when debug logging is enabled with `debug` == `true` in the client options) denoting why the transaction was dropped.
+
+This option must default to an empty sequence if it's introduced in a release with a minor SemVer bump.
+SDKs should set the default for this option to `[404]` at the earliest release with a major SemVer bump following its introduction.
```

Comment:
we currently default to the following range of ignored status codes: `[[401, 404], [301, 303], [305, 399]]`.

For example, users often complain about traced redirect calls for moved resources. Similarly getting permission denied errors/forbidden (401/403) for bots sending requests to protected endpoints is equally spammy as 404s. I think we want to be a bit more aggressive in terms of default values but happy to leave go with slimmer defaults if you/other reviewers prefer.